src/test/java/org/jenkinsci/plugins/workflow/CpsStringGroovyMethodsTest.java

```diff
@@ -0,0 +1,67 @@
+package org.jenkinsci.plugins.workflow;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class CpsStringGroovyMethodsTest {
+    @ClassRule
+    public static JenkinsRule r = new JenkinsRule();
```

Comment:
Ahyup. Since this is all purely self contained tests with no side effects, why not?